docs/.vitepress/config.js

```diff
@@ -51,6 +51,7 @@ export default defineConfig({
           { text: 'Quick start', link: '/getting-started/quick-start' },
           { text: 'Simple TODO app', link: '/getting-started/simple-todo-app' },
           { text: 'How it works', link: '/getting-started/how-it-works' },
+          { text: 'Vite in SFX', link: '/getting-started/vite-in-sfx' },
```

Comment:
I would put it into the Introduction chapter. The whole introduction should contain section referencing other technologies (VIte, Nuxt, ...) with explenations why/why not.